catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesLuceneIndexer.java

```diff
@@ -229,10 +228,10 @@ private void addDocument(final IndexWriter indexWriter, final GeoEntry geoEntry,
                 geoEntry.getName(),
                 Field.Store.YES));
 
-        document.add(new DoubleField(GeoNamesLuceneConstants.LATITUDE_FIELD,
+        document.add(new LegacyDoubleField(GeoNamesLuceneConstants.LATITUDE_FIELD,
```

Comment:
Is there a reason you could not use `DoublePoint`?

https://lucene.apache.org/core/6_0_0/core/org/apache/lucene/document/DoublePoint.html
https://lucene.apache.org/core/6_0_0/core/org/apache/lucene/document/LegacyDoubleField.html